### ISSUE TITLE
Fixing alignment of actions area

### ISSUE THREAD

**Issue by arfon** [openjournals/theoj#274]
Right now there's a small gap between the right-hand column and the actions area. Could this be closed up? I'm struggling to do this.

![screen_shot_2016-12-30_at_7_54_12_am](https://cloud.githubusercontent.com/assets/4483/21565408/7e027cc8-ce65-11e6-9bf6-33a2df1529b7.png)

/ cc @marcrohloff

**Comment by marcrohloff** [comment]
Yes, I can fix that but do you want to? The gray is the page background.
(For ex you can see it between pages). It used to be white but you changed
tat in one of your commits
​

**Comment by arfon** [comment]
> Yes, I can fix that but do you want to?

Yeah, I think so. It looks like it's part of the content of the right-hand column in David's designs.

>  It used to be white but you changed that in one of your commits

Yeah, this was to reflect David's designs too.
​

**Comment by marcrohloff** [comment]
I brought this up a couple of weeks back.
David's design has no space between the page and the right hand column. It
looks great if there is just one page, but it doesn't work with the current
multiple page design. Each page has a shadow to make it look like a page.
This is completely lost by not having a background for the pages.

Marc
​

**Comment by arfon** [comment]
Oh ok. Sorry I missed this context.

I think we should ask David to address this in a new design iteration.
